src/components/timebank/timebank-content.tsx

```diff
@@ -164,125 +167,170 @@ const TimebankContent = (props: Props) => {
 
   return (
     <>
-      <Card sx={{ p: 3 }}>
-        <Typography gutterBottom variant="h5" sx={{ textAlign: "center" }}>
-          {strings.timebank.barChartDescription}
-        </Typography>
-        <Box sx={{ display: "flex", justifyContent: "center" }}>
-          <Select
+      <Grow in>
+        <Card sx={{ border: "2px solid #bdbdbd;" }}>
+          <Typography
+            gutterBottom
+            fontWeight={"bold"}
+            variant="h6"
             sx={{
-              width: "50%",
-              marginRight: "1%",
-              textAlign: "center"
-            }}
-            value={timespan}
-            onChange={(e) => {
-              setTimespan(e.target.value as Timespan);
+              color: "white",
+              textAlign: "center",
+              backgroundColor: "#bdbdbd",
+              width: "100%",
+              p: 2
             }}
           >
-            <MenuItem value={Timespan.WEEK}>{strings.timeExpressions.week}</MenuItem>
-            <MenuItem value={Timespan.MONTH}>{strings.timeExpressions.month}</MenuItem>
-            <MenuItem value={Timespan.ALL_TIME}>{strings.timeExpressions.allTime}</MenuItem>
-          </Select>
-        </Box>
-        <Box sx={{ display: "flex", flexDirection: "row", justifyItems: "center" }}>
-          {renderOverViewChart()}
-        </Box>
-      </Card>
+            {strings.timebank.barChartDescription}
+          </Typography>
+          <Container sx={{ p: 3 }}>
+            <ListItemText
+              sx={{ textAlign: "center", scale: "150%", mb: 3 }}
```

Comment:
As a general rule 3 or more properties in a line and it can be worth starting a new line for each one, so:
textAlign: "center",
scale: "150%", etc